contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViews.java

```diff
@@ -138,6 +138,18 @@
           RpcViewConstants.GRPC_SERVER_SENT_MESSAGES_PER_METHOD_VIEW,
           RpcViewConstants.GRPC_SERVER_RECEIVED_MESSAGES_PER_METHOD_VIEW);
 
+  @VisibleForTesting
+  static final ImmutableSet<View> GRPC_CLIENT_BASIC_VIEWS_SET =
+      ImmutableSet.of(
+          RpcViewConstants.GRPC_CLIENT_ROUNDTRIP_LATENCY_VIEW,
```

Comment:
Currently gRPC is still recording against the old constants (`RPC_CLIENT_ROUNDTRIP_LATENCY_VIEW`, etc.). Consider including them in the set too.

Reply:
Will gRPC record against both (old and new) when it transitions? If yes, then it will double the metrics.

Reply:
added old constants.

Reply:
> Will gRPC record against both (old and new) when it transitions? 

No, when it transitions gRPC will only record against the new constants (recording against both has caused performance regression for them).